Gemfile

```diff
@@ -17,7 +17,7 @@ gem 'jbuilder', '~> 2.5'
 gem 'bootsnap'
 gem 'faraday'
 gem 'figaro'
-
+gem 'dotenv-rails'
```

Comment:
👀  I love seeing when y'all try out tools other than the ones we talk about in class. Nice!